modules/apps/frontend-js/frontend-js-test/src/testFunctional/A11yTool.testcase

```diff
@@ -0,0 +1,51 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "User Interface";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		property osgi.modules.includes = "frontend-js-clay-sample-web";
+
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Classic Theme Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Guest",
+			layoutName = "Classic Theme Page",
+			widgetName = "JS Clay Sample Web");
+
+		Navigator.gotoPage(pageName = "Classic Theme Page");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			Page.tearDownCP();
+		}
+	}
+
+	@description = "LPS-140736. Ensures there is no accessibility side panel on a page with no violations."
+	@priority = "5"
+	test CanDetectNoViolations {
+		Navigator.gotoPage(pageName = "Home");
+
+		if (IsElementPresent(locator1 = "A11yTool#ACCESSIBILITY_PANEL_HEADING")) {
+			Refresh();
+		}
+
+		AssertElementNotPresent(locator1 = "A11yTool#ACCESSIBILITY_PANEL_HEADING");
```

Comment:
You have the first assertion but remember the other two assertions from the test case.

```
Then side panel will list no violations on the page
And Then there are no highlight elements present on the page
And Then there are no icons present on the page ("!" icon next to potential highlight element)
```